Binance.Net/Objects/Spot/SubAccountData/BinanceSubAccount.cs

```diff
@@ -9,9 +9,6 @@ namespace Binance.Net.Objects.Spot.SubAccountData
 {
     internal class BinanceSubAccountWrapper
     {
-        [JsonProperty("msg")]
-        public string? Message { get; set; }
-        public bool Success { get; set; }
```

Comment:
In my test it also returned `"success": true`. Even though official documentation doesn't have this field anymore I would rather leave both Success and Message

Reply:
This is response that Binance currently returns. Many fields were removed and `IsFreeze` was added to `SubAccounts` 
```
{
    "subAccounts": [
        {
            "email": "test@test.com",
            "isFreeze": false,
            "createTime": 1606813254000
        }
    ],
    "success": true
}
```

Reply:
Great, will remove other properties and Account Status filter

Reply:
I can add Success property to BinanceSubAccountWrapper, but I dont think it is correct to check if result.Success is true, since it is not part of the official documentation. IMO they can change it from 1st of August too, and then code won't work. 

Not sure what is correct approach.

Reply:
Hm yes that's a good question. I think we just leave the response handling as it is, which means not explicitly checking the success flag, but instead relying on that Binance will send a non-success Http status code when there is something wrong. 

This can be evaluated after the 1st of August.